==3943089== Memcheck, a memory error detector
==3943089== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3943089== Using Valgrind-3.18.1-42b08ed5bd-20211015 and LibVEX; rerun with -h for copyright info
==3943089== Command: java -Djava.library.path=/home/crow/git/arb4j/bin/.. -da -classpath /home/crow/.m2/repository/org/openjfx/javafx-graphics/20-ea+9/javafx-graphics-20-ea+9.jar:/home/crow/.m2/repository/org/openjfx/javafx-graphics/20-ea+9/javafx-graphics-20-ea+9-linux.jar:/home/crow/.m2/repository/org/openjfx/javafx-base/20-ea+9/javafx-base-20-ea+9.jar:/home/crow/.m2/repository/org/openjfx/javafx-base/20-ea+9/javafx-base-20-ea+9-linux.jar:/home/crow/.m2/repository/com/massisframework/j-text-utils/0.3.4/j-text-utils-0.3.4.jar:/home/crow/.m2/repository/com/google/guava/guava/21.0/guava-21.0.jar:/home/crow/.m2/repository/commons-lang/commons-lang/2.6/commons-lang-2.6.jar:/home/crow/.m2/repository/au/com/bytecode/opencsv/2.4/opencsv-2.4.jar:/home/crow/.m2/repository/org/ow2/asm/asm/9.3/asm-9.3.jar:/home/crow/.m2/repository/org/ow2/asm/asm-util/9.3/asm-util-9.3.jar:/home/crow/.m2/repository/org/ow2/asm/asm-tree/9.3/asm-tree-9.3.jar:/home/crow/.m2/repository/org/ow2/asm/asm-analysis/9.3/asm-analysis-9.3.jar:/home/crow/.m2/repository/io/humble/humble-video-all/0.3.0/humble-video-all-0.3.0.jar:/home/crow/.m2/repository/io/humble/humble-video-noarch/0.3.0/humble-video-noarch-0.3.0.jar:/home/crow/.m2/repository/io/humble/humble-video-arch-x86_64-pc-linux-gnu6/0.3.0/humble-video-arch-x86_64-pc-linux-gnu6-0.3.0.jar:/home/crow/.m2/repository/org/scilab/forge/jlatexmath/1.0.5/jlatexmath-1.0.5.jar:/home/crow/.m2/repository/org/scilab/forge/jlatexmath-font-greek/1.0.5/jlatexmath-font-greek-1.0.5.jar:/home/crow/.m2/repository/org/scilab/forge/jlatexmath-font-cyrillic/1.0.5/jlatexmath-font-cyrillic-1.0.5.jar:/home/crow/.m2/repository/org/l33tlabs/twl/pngdecoder/1.0/pngdecoder-1.0.jar:/home/crow/.m2/repository/de/gsi/chart/chartfx-chart/11.2.7/chartfx-chart-11.2.7.jar:/home/crow/.m2/repository/org/openjfx/javafx-controls/13.0.1/javafx-controls-13.0.1.jar:/home/crow/.m2/repository/org/openjfx/javafx-controls/13.0.1/javafx-controls-13.0.1-linux.jar:/home/crow/.m2/repository/org/kordamp/ikonli/ikonli-javafx/11.5.0/ikonli-javafx-11.5.0.jar:/home/crow/.m2/repository/org/kordamp/ikonli/ikonli-core/11.5.0/ikonli-core-11.5.0.jar:/home/crow/.m2/repository/org/kordamp/ikonli/ikonli-fontawesome-pack/11.5.0/ikonli-fontawesome-pack-11.5.0.jar:/home/crow/.m2/repository/org/kordamp/ikonli/ikonli-fontawesome5-pack/11.5.0/ikonli-fontawesome5-pack-11.5.0.jar:/home/crow/.m2/repository/de/gsi/dataset/chartfx-dataset/11.2.7/chartfx-dataset-11.2.7.jar:/home/crow/.m2/repository/it/unimi/dsi/fastutil/8.3.1/fastutil-8.3.1.jar:/home/crow/.m2/repository/com/ibm/icu/icu4j/65.1/icu4j-65.1.jar:/home/crow/.m2/repository/ar/com/hjg/pngj/2.1.0/pngj-2.1.0.jar:/home/crow/.m2/repository/org/slf4j/slf4j-api/2.0.0-alpha0/slf4j-api-2.0.0-alpha0.jar:/home/crow/.m2/repository/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar:/home/crow/.m2/repository/org/jetbrains/annotations/21.0.1/annotations-21.0.1.jar:/home/crow/.m2/repository/de/gsi/math/chartfx-math/11.2.7/chartfx-math-11.2.7.jar:/home/crow/.m2/repository/com/github/wendykierp/JTransforms/3.1/JTransforms-3.1.jar:/home/crow/.m2/repository/pl/edu/icm/JLargeArrays/1.5/JLargeArrays-1.5.jar:/home/crow/.m2/repository/org/apache/commons/commons-math3/3.6.1/commons-math3-3.6.1.jar:/home/crow/.m2/repository/org/controlsfx/controlsfx/11.1.1/controlsfx-11.1.1.jar:/home/crow/.m2/repository/junit/junit/4.13.2/junit-4.13.2.jar:/home/crow/.m2/repository/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar:/home/crow/.m2/repository/org/jogamp/jogl/jogl-all/2.3.1/jogl-all-2.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-assimp/3.3.1/lwjgl-assimp-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/joml/joml/1.10.4/joml-1.10.4.jar:/home/crow/.m2/repository/com/github/orange451/LWJGUI/pre-mt/LWJGUI-pre-mt.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-nanovg/3.3.1/lwjgl-nanovg-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-nanovg/3.3.1/lwjgl-nanovg-3.3.1-natives-linux.jar:/home/crow/.m2/repository/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1-natives-linux.jar:/home/crow/git/arb4j/bin/../target/classes -ea --module-path=/usr/share/java/javafx-jmods-19 --enable-preview --enable-native-access=arb4j --add-modules=javafx.graphics --add-reads javafx.graphics=ALL-UNNAMED --add-opens javafx.controls/com.sun.javafx.charts=ALL-UNNAMED --add-opens javafx.controls/com.sun.javafx.scene.control.inputmap=ALL-UNNAMED --add-opens javafx.graphics/com.sun.javafx.iio=ALL-UNNAMED --add-opens javafx.graphics/com.sun.javafx.iio.common=ALL-UNNAMED --add-opens javafx.graphics/com.sun.javafx.css=ALL-UNNAMED --add-opens javafx.base/com.sun.javafx.runtime=ALL-UNNAMED --add-exports javafx.controls/com.sun.javafx.scene.control.behavior=ALL-UNNAMED --add-exports javafx.graphics/com.sun.javafx.scene.traversal=ALL-UNNAMED arb.viz.AnimatedComplexFunctionSequencer
==3943089== Parent PID: 3943086
==3943089== 
--3943089-- 
--3943089-- Valgrind options:
--3943089--    --leak-check=full
--3943089--    --show-leak-kinds=all
--3943089--    --suppressions=valgrind.suppress
--3943089--    --log-file=valgrind.log
--3943089--    -v
--3943089-- Contents of /proc/version:
--3943089--   Linux version 5.19.0-23-generic (buildd@lcy02-amd64-076) (x86_64-linux-gnu-gcc-12 (Ubuntu 12.2.0-3ubuntu1) 12.2.0, GNU ld (GNU Binutils for Ubuntu) 2.39) #24-Ubuntu SMP PREEMPT_DYNAMIC Fri Oct 14 15:39:57 UTC 2022
--3943089-- 
--3943089-- Arch and hwcaps: AMD64, LittleEndian, amd64-cx16-lzcnt-rdtscp-sse3-ssse3-avx-avx2-bmi-f16c-rdrand-rdseed
--3943089-- Page sizes: currently 4096, max supported 4096
--3943089-- Valgrind library directory: /usr/libexec/valgrind
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/bin/java
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
--3943089--   Considering /usr/lib/debug/.build-id/29/2e105c0bb3ee8e8f5b917f8af764373d206659.debug ..
--3943089--   .. build-id is valid
--3943089-- Reading syms from /usr/libexec/valgrind/memcheck-amd64-linux
--3943089--   Considering /usr/lib/debug/.build-id/84/fedd9a4c04941b8d123398ad0dd0ebc25c120c.debug ..
--3943089--   .. build-id is valid
--3943089--    object doesn't have a dynamic symbol table
--3943089-- Scheduler: using generic scheduler lock implementation.
--3943089-- Reading suppressions file: valgrind.suppress
--3943089-- Reading suppressions file: /usr/libexec/valgrind/default.supp
==3943089== embedded gdbserver: reading from /tmp/vgdb-pipe-from-vgdb-to-3943089-by-crow-on-???
==3943089== embedded gdbserver: writing to   /tmp/vgdb-pipe-to-vgdb-from-3943089-by-crow-on-???
==3943089== embedded gdbserver: shared mem   /tmp/vgdb-pipe-shared-mem-vgdb-3943089-by-crow-on-???
==3943089== 
==3943089== TO CONTROL THIS PROCESS USING vgdb (which you probably
==3943089== don't want to do, unless you know exactly what you're doing,
==3943089== or are doing some strange experiment):
==3943089==   /usr/bin/vgdb --pid=3943089 ...command...
==3943089== 
==3943089== TO DEBUG THIS PROCESS USING GDB: start GDB like this
==3943089==   /path/to/gdb java
==3943089== and then give GDB the following command
==3943089==   target remote | /usr/bin/vgdb --pid=3943089
==3943089== --pid is optional if only one valgrind process is running
==3943089== 
--3943089-- REDIR: 0x4026dd0 (ld-linux-x86-64.so.2:strlen) redirected to 0x580bcec2 (vgPlain_amd64_linux_REDIR_FOR_strlen)
--3943089-- REDIR: 0x4025580 (ld-linux-x86-64.so.2:index) redirected to 0x580bcedc (vgPlain_amd64_linux_REDIR_FOR_index)
--3943089-- Reading syms from /usr/libexec/valgrind/vgpreload_core-amd64-linux.so
--3943089--   Considering /usr/lib/debug/.build-id/7b/e1c1588ba6713462a4091c80bbb78004f47ba6.debug ..
--3943089--   .. build-id is valid
--3943089-- Reading syms from /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so
--3943089--   Considering /usr/lib/debug/.build-id/ca/411cf3a4b32c9d21bbc464935b0f1af28fb2d8.debug ..
--3943089--   .. build-id is valid
==3943089== WARNING: new redirection conflicts with existing -- ignoring it
--3943089--     old: 0x04026dd0 (strlen              ) R-> (0000.0) 0x580bcec2 vgPlain_amd64_linux_REDIR_FOR_strlen
--3943089--     new: 0x04026dd0 (strlen              ) R-> (2007.0) 0x0484ae30 strlen
--3943089-- REDIR: 0x40257b0 (ld-linux-x86-64.so.2:strcmp) redirected to 0x484bcd0 (strcmp)
--3943089-- REDIR: 0x40249d0 (ld-linux-x86-64.so.2:mempcpy) redirected to 0x484f840 (mempcpy)
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
--3943089--    object doesn't have a symbol table
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libjli.so
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/libpthread.so.0
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/libdl.so.2
--3943089--   Considering /usr/lib/debug/.build-id/bb/9bae4bb2a97b1c469b3fcf60b636982d448160.debug ..
--3943089--   .. build-id is valid
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/libc.so.6
--3943089--   Considering /usr/lib/debug/.build-id/d1/704d25fbbb72fa95d517b883131828c0883fe9.debug ..
--3943089--   .. build-id is valid
==3943089== WARNING: new redirection conflicts with existing -- ignoring it
--3943089--     old: 0x049601d0 (memalign            ) R-> (1011.0) 0x0484a080 memalign
--3943089--     new: 0x049601d0 (memalign            ) R-> (1017.0) 0x0484a050 aligned_alloc
==3943089== WARNING: new redirection conflicts with existing -- ignoring it
--3943089--     old: 0x049601d0 (memalign            ) R-> (1011.0) 0x0484a080 memalign
--3943089--     new: 0x049601d0 (memalign            ) R-> (1017.0) 0x0484a020 aligned_alloc
--3943089-- REDIR: 0x4967040 (libc.so.6:strnlen) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49670d0 (libc.so.6:strpbrk) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49651a0 (libc.so.6:strcmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497c660 (libc.so.6:wcsnlen) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49642b0 (libc.so.6:memset) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497ad30 (libc.so.6:wcslen) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4963790 (libc.so.6:memcpy@@GLIBC_2.14) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497ab40 (libc.so.6:wcschr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4965090 (libc.so.6:index) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4967100 (libc.so.6:rindex) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497abd0 (libc.so.6:wcscmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964500 (libc.so.6:stpncpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497b180 (libc.so.6:wmemchr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966ee0 (libc.so.6:strncmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964570 (libc.so.6:strcasecmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49663f0 (libc.so.6:strcspn) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x497ac60 (libc.so.6:wcscpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4965010 (libc.so.6:strcat) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966dd0 (libc.so.6:strncasecmp_l) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4963690 (libc.so.6:bcmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964220 (libc.so.6:memrchr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4965110 (libc.so.6:strchrnul) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966370 (libc.so.6:strcpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964610 (libc.so.6:strcasecmp_l) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966ca0 (libc.so.6:strlen) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966f80 (libc.so.6:strncpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964480 (libc.so.6:stpcpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4963fe0 (libc.so.6:memmove) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
==3943089== Preferring higher priority redirection:
--3943089--     old: 0x04a30c00 (__memcpy_avx_unalign) R-> (2018.0) 0x0484cf90 __memcpy_avx_unaligned_erms
--3943089--     new: 0x04a30c00 (__memcpy_avx_unalign) R-> (2018.1) 0x0484e880 memmove
--3943089-- REDIR: 0x4963600 (libc.so.6:memchr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49672c0 (libc.so.6:strspn) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4964100 (libc.so.6:mempcpy) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4966d30 (libc.so.6:strncasecmp) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x49643b0 (libc.so.6:rawmemchr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4967af0 (libc.so.6:strstr) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4a36cf0 (libc.so.6:__strrchr_avx2) redirected to 0x484a810 (rindex)
--3943089-- REDIR: 0x495f660 (libc.so.6:malloc) redirected to 0x4844820 (malloc)
--3943089-- REDIR: 0x4a343a0 (libc.so.6:__strlen_avx2) redirected to 0x484ad10 (strlen)
--3943089-- REDIR: 0x4a30c00 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x484e880 (memmove)
--3943089-- REDIR: 0x4a35a70 (libc.so.6:__strncmp_avx2) redirected to 0x484b3e0 (strncmp)
--3943089-- REDIR: 0x4a33ab0 (libc.so.6:__strcmp_avx2) redirected to 0x484bbd0 (strcmp)
--3943089-- REDIR: 0x4a338c0 (libc.so.6:__strchrnul_avx2) redirected to 0x484f330 (strchrnul)
--3943089-- REDIR: 0x4a30bc0 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x484f440 (mempcpy)
--3943089-- REDIR: 0x497a0b0 (libc.so.6:__strstr_sse2_unaligned) redirected to 0x484fa50 (strstr)
--3943089-- REDIR: 0x4a33680 (libc.so.6:__strchr_avx2) redirected to 0x484a9f0 (index)
--3943089-- REDIR: 0x4a30200 (libc.so.6:__memchr_avx2) redirected to 0x484bd50 (memchr)
--3943089-- REDIR: 0x495f970 (libc.so.6:free) redirected to 0x4847210 (free)
--3943089-- REDIR: 0x4a50740 (libc.so.6:__strcspn_sse42) redirected to 0x484fc70 (strcspn)
--3943089-- REDIR: 0x4a533f0 (libc.so.6:__strspn_sse42) redirected to 0x484fd50 (strspn)
--3943089-- REDIR: 0x4a360c0 (libc.so.6:__strncpy_avx2) redirected to 0x484afe0 (strncpy)
--3943089-- REDIR: 0x4960a60 (libc.so.6:calloc) redirected to 0x48499d0 (calloc)
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/server/libjvm.so
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/librt.so.1
--3943089--   Considering /usr/lib/debug/.build-id/a7/e413886a78c7b9693488186f9193464909137a.debug ..
--3943089--   .. build-id is valid
--3943089-- Reading syms from /usr/lib/x86_64-linux-gnu/libm.so.6
--3943089--   Considering /usr/lib/debug/.build-id/69/c0f56e574b311a0d256e8ede640fab11c1407d.debug ..
--3943089--   .. build-id is valid
--3943089-- REDIR: 0x4966e70 (libc.so.6:strncat) redirected to 0x483b220 (_vgnU_ifunc_wrapper)
--3943089-- REDIR: 0x4a31600 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x484e770 (memset)
--3943089-- REDIR: 0x4a34010 (libc.so.6:__strcpy_avx2) redirected to 0x484ae60 (strcpy)
--3943089-- REDIR: 0x496caf0 (libc.so.6:memcpy@GLIBC_2.2.5) redirected to 0x484be00 (memcpy@GLIBC_2.2.5)
--3943089-- REDIR: 0x4a325f0 (libc.so.6:__strcasecmp_avx2) redirected to 0x484b560 (strcasecmp)
--3943089-- REDIR: 0x4a304a0 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x484e010 (bcmp)
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libjimage.so
--3943089-- REDIR: 0x495fcd0 (libc.so.6:realloc) redirected to 0x4849c50 (realloc)
--3943089-- REDIR: 0x4a31740 (libc.so.6:__rawmemchr_avx2) redirected to 0x484f360 (rawmemchr)
--3943089-- REDIR: 0x4a35060 (libc.so.6:__strncat_avx2) redirected to 0x484abc0 (strncat)
--3943089-- REDIR: 0x4a34520 (libc.so.6:__strncasecmp_avx2) redirected to 0x484b640 (strncasecmp)
--3943089-- REDIR: 0x4a36a20 (libc.so.6:__strnlen_avx2) redirected to 0x484acb0 (strnlen)
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libjava.so
==3943089== Warning: set address range perms: large range [0x417800000, 0x800000000) (noaccess)
==3943089== Warning: set address range perms: large range [0x417800000, 0x456800000) (defined)
==3943089== Warning: set address range perms: large range [0x800000000, 0x841000000) (noaccess)
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libjsvml.so
==3943089== Thread 2:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6416548 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6415548 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6414548 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6413548 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6412548 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6411548 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64164e0 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64154e0 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64144e0 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64134e0 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64124e0 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64114e0 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6416480 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6415480 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6414480 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6413480 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6412480 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6411480 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6416460 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6415460 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6414460 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6413460 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12E4: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6412460 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12EB: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6411460 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6415ce0 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6414ce0 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6413ce0 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6412ce0 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12E4: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6411ce0 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12EB: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6410ce0 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6414c50 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6413c50 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6412c50 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6411c50 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6410c50 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64142a8 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64132a8 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64122a8 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64112a8 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6413d78 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6412d78 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6411d78 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6410d78 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6413d18 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6412d18 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6411d18 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6410d18 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libnio.so
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA0: strncmp (strcmp-sse2.S:160)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x4006456: _dl_dst_count (dl-load.c:253)
==3943089==    by 0x4006456: expand_dynamic_string_token (dl-load.c:395)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e1 is 1 bytes inside a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA8: strncmp (strcmp-sse2.S:162)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x4006456: _dl_dst_count (dl-load.c:253)
==3943089==    by 0x4006456: expand_dynamic_string_token (dl-load.c:395)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e9 is 1 bytes after a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA0: strncmp (strcmp-sse2.S:160)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x40062C5: _dl_dst_substitute (dl-load.c:295)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e1 is 1 bytes inside a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA8: strncmp (strcmp-sse2.S:162)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x40062C5: _dl_dst_substitute (dl-load.c:295)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e9 is 1 bytes after a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
--3943089-- REDIR: 0x4024ec0 (ld-linux-x86-64.so.2:stpcpy) redirected to 0x484e4f0 (stpcpy)
--3943089-- Reading syms from /usr/lib/jvm/jdk-19/lib/libnet.so
==3943089== 
==3943089== HEAP SUMMARY:
==3943089==     in use at exit: 35,486,948 bytes in 5,759 blocks
==3943089==   total heap usage: 8,429 allocs, 2,670 frees, 35,923,309 bytes allocated
==3943089== 
==3943089== Searching for pointers to 5,759 not-freed blocks
==3943089== Checked 1,181,053,368 bytes
==3943089== 
==3943089== Thread 1:
==3943089== 5 bytes in 1 blocks are still reachable in loss record 12 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x496643D: strdup (strdup.c:42)
==3943089==    by 0x48AB1B8: JLI_StringDup (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x1089F5: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 16 bytes in 1 blocks are still reachable in loss record 112 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x400690B: malloc (rtld-malloc.h:56)
==3943089==    by 0x400690B: decompose_rpath (dl-load.c:644)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089== 
==3943089== 16 bytes in 2 blocks are still reachable in loss record 122 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x496643D: strdup (strdup.c:42)
==3943089==    by 0x48AB1B8: JLI_StringDup (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A73D6: ReadKnownVMs (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AA2DD: CreateExecutionEnvironment (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7D16: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 24 bytes in 1 blocks are still reachable in loss record 280 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AB3A3: JLI_List_new (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x1089E9: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 29 bytes in 1 blocks are still reachable in loss record 427 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x496643D: strdup (strdup.c:42)
==3943089==    by 0x48AB1B8: JLI_StringDup (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AA241: SetExecname (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AA298: CreateExecutionEnvironment (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7D16: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 34 bytes in 1 blocks are still reachable in loss record 628 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4005AE0: malloc (rtld-malloc.h:56)
==3943089==    by 0x4005AE0: open_path (dl-load.c:1926)
==3943089==    by 0x4008B49: _dl_map_object (dl-load.c:2116)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089== 
==3943089== 60 bytes in 1 blocks are still reachable in loss record 1,194 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7248: SetJavaCommandLineProp (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A8EEC: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 64 bytes in 1 blocks are still reachable in loss record 1,195 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x485B527: ImageFileReaderTable::ImageFileReaderTable() (in /usr/lib/jvm/jdk-19/lib/libjimage.so)
==3943089==    by 0x485A70F: _GLOBAL__sub_I_imageFile.cpp (in /usr/lib/jvm/jdk-19/lib/libjimage.so)
==3943089==    by 0x4004FBD: call_init.part.0 (dl-init.c:70)
==3943089==    by 0x40050A7: call_init (dl-init.c:33)
==3943089==    by 0x40050A7: _dl_init (dl-init.c:117)
==3943089==    by 0x4A2C7F3: _dl_catch_exception (dl-error-skeleton.c:182)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:808)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:771)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089== 
==3943089== 65 bytes in 2 blocks are still reachable in loss record 1,336 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40144FD: _dl_load_cache_lookup (dl-cache.c:524)
==3943089==    by 0x4008F99: _dl_map_object (dl-load.c:2142)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 99 bytes in 3 blocks are still reachable in loss record 1,454 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x400BB98: malloc (rtld-malloc.h:56)
==3943089==    by 0x400BB98: _dl_new_object (dl-object.c:199)
==3943089==    by 0x4006F9E: _dl_map_object_from_fd (dl-load.c:1063)
==3943089==    by 0x4008A3F: _dl_map_object (dl-load.c:2275)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 160 bytes in 1 blocks are still reachable in loss record 1,688 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x400BFEB: malloc (rtld-malloc.h:56)
==3943089==    by 0x400BFEB: add_to_global_resize (dl-open.c:152)
==3943089==    by 0x400CF97: dl_open_worker_begin (dl-open.c:716)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089==    by 0x494A93E: _dlerror_run (dlerror.c:138)
==3943089==    by 0x494AF20: dlopen_implementation (dlopen.c:71)
==3943089==    by 0x494AF20: dlopen@@GLIBC_2.34 (dlopen.c:81)
==3943089== 
==3943089== 183 bytes in 5 blocks are still reachable in loss record 1,713 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40089BA: _dl_map_object (dl-load.c:2208)
==3943089==    by 0x400C928: dl_open_worker_begin (dl-open.c:534)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089==    by 0x494A93E: _dlerror_run (dlerror.c:138)
==3943089== 
==3943089== 183 bytes in 5 blocks are still reachable in loss record 1,714 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x400BB98: malloc (rtld-malloc.h:56)
==3943089==    by 0x400BB98: _dl_new_object (dl-object.c:199)
==3943089==    by 0x4006F9E: _dl_map_object_from_fd (dl-load.c:1063)
==3943089==    by 0x4008A3F: _dl_map_object (dl-load.c:2275)
==3943089==    by 0x400C928: dl_open_worker_begin (dl-open.c:534)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089== 
==3943089== 240 bytes in 1 blocks are still reachable in loss record 1,778 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AB3B7: JLI_List_new (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x1089E9: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 240 bytes in 1 blocks are possibly lost in loss record 1,803 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A6985: CheckJvmType (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AA2FB: CreateExecutionEnvironment (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7D16: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 288 bytes in 1 blocks are possibly lost in loss record 1,837 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x4012949: calloc (rtld-malloc.h:44)
==3943089==    by 0x4012949: allocate_dtv (dl-tls.c:375)
==3943089==    by 0x4012949: _dl_allocate_tls (dl-tls.c:634)
==3943089==    by 0x494FF99: allocate_stack (allocatestack.c:423)
==3943089==    by 0x494FF99: pthread_create@@GLIBC_2.34 (pthread_create.c:650)
==3943089==    by 0x48AA81D: CallJavaMainInNewThread (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A794C: ContinueInNewThread (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A8F2F: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 320 bytes in 1 blocks are still reachable in loss record 1,859 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7491: ReadKnownVMs (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48AA2DD: CreateExecutionEnvironment (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A7D16: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 512 bytes in 1 blocks are still reachable in loss record 1,901 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A6EC6: AddOption (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A91D1: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 576 bytes in 9 blocks are still reachable in loss record 1,922 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A91A6: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 1,944 bytes in 8 blocks are still reachable in loss record 2,012 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x4013CA3: calloc (rtld-malloc.h:44)
==3943089==    by 0x4013CA3: _dl_check_map_versions (dl-version.c:280)
==3943089==    by 0x400CB93: dl_open_worker_begin (dl-open.c:600)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089==    by 0x494A93E: _dlerror_run (dlerror.c:138)
==3943089==    by 0x494AF20: dlopen_implementation (dlopen.c:71)
==3943089==    by 0x494AF20: dlopen@@GLIBC_2.34 (dlopen.c:81)
==3943089== 
==3943089== 3,679 bytes in 3 blocks are still reachable in loss record 2,033 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x400B89C: calloc (rtld-malloc.h:44)
==3943089==    by 0x400B89C: _dl_new_object (dl-object.c:92)
==3943089==    by 0x4006F9E: _dl_map_object_from_fd (dl-load.c:1063)
==3943089==    by 0x4008A3F: _dl_map_object (dl-load.c:2275)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 4,900 bytes in 1 blocks are still reachable in loss record 2,041 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x48AB168: JLI_MemAlloc (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A858A: JLI_Launch (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AD7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 5,688 bytes in 28 blocks are still reachable in loss record 2,051 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x496643D: strdup (strdup.c:42)
==3943089==    by 0x48AB1B8: JLI_StringDup (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x108AF7: main (in /usr/lib/jvm/jdk-19/bin/java)
==3943089== 
==3943089== 6,263 bytes in 5 blocks are still reachable in loss record 2,054 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x400B89C: calloc (rtld-malloc.h:44)
==3943089==    by 0x400B89C: _dl_new_object (dl-object.c:92)
==3943089==    by 0x4006F9E: _dl_map_object_from_fd (dl-load.c:1063)
==3943089==    by 0x4008A3F: _dl_map_object (dl-load.c:2275)
==3943089==    by 0x400C928: dl_open_worker_begin (dl-open.c:534)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089== 
==3943089== 49,152 bytes in 1 blocks are still reachable in loss record 2,097 of 2,110
==3943089==    at 0x4849A83: calloc (vg_replace_malloc.c:1328)
==3943089==    by 0x47330EDB: init (in /usr/lib/jvm/jdk-19/lib/libnet.so)
==3943089==    by 0x4004FBD: call_init.part.0 (dl-init.c:70)
==3943089==    by 0x40050A7: call_init (dl-init.c:33)
==3943089==    by 0x40050A7: _dl_init (dl-init.c:117)
==3943089==    by 0x4A2C7F3: _dl_catch_exception (dl-error-skeleton.c:182)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:808)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:771)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089==    by 0x494A93E: _dlerror_run (dlerror.c:138)
==3943089== 
==3943089== 72,704 bytes in 1 blocks are still reachable in loss record 2,101 of 2,110
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x485A759: _GLOBAL__sub_I_eh_alloc.cc (in /usr/lib/jvm/jdk-19/lib/libjimage.so)
==3943089==    by 0x4004FBD: call_init.part.0 (dl-init.c:70)
==3943089==    by 0x40050A7: call_init (dl-init.c:33)
==3943089==    by 0x40050A7: _dl_init (dl-init.c:117)
==3943089==    by 0x4A2C7F3: _dl_catch_exception (dl-error-skeleton.c:182)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:808)
==3943089==    by 0x400BE95: dl_open_worker (dl-open.c:771)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089==    by 0x494AE6B: dlopen_doit (dlopen.c:56)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x4A2C852: _dl_catch_error (dl-error-skeleton.c:227)
==3943089==    by 0x494A93E: _dlerror_run (dlerror.c:138)
==3943089== 
==3943089== LEAK SUMMARY:
==3943089==    definitely lost: 0 bytes in 0 blocks
==3943089==    indirectly lost: 0 bytes in 0 blocks
==3943089==      possibly lost: 528 bytes in 2 blocks
==3943089==    still reachable: 146,916 bytes in 84 blocks
==3943089==         suppressed: 35,339,504 bytes in 5,673 blocks
==3943089== 
==3943089== ERROR SUMMARY: 118 errors from 53 contexts (suppressed: 60 from 60)
==3943089== 
==3943089== 1 errors in context 1 of 53:
==3943089== Thread 2:
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA8: strncmp (strcmp-sse2.S:162)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x40062C5: _dl_dst_substitute (dl-load.c:295)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e9 is 1 bytes after a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 
==3943089== 1 errors in context 2 of 53:
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA0: strncmp (strcmp-sse2.S:160)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x40062C5: _dl_dst_substitute (dl-load.c:295)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e1 is 1 bytes inside a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 
==3943089== 1 errors in context 3 of 53:
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA8: strncmp (strcmp-sse2.S:162)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x4006456: _dl_dst_count (dl-load.c:253)
==3943089==    by 0x4006456: expand_dynamic_string_token (dl-load.c:395)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e9 is 1 bytes after a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 
==3943089== 1 errors in context 4 of 53:
==3943089== Invalid read of size 8
==3943089==    at 0x4026FA0: strncmp (strcmp-sse2.S:160)
==3943089==    by 0x40051CE: is_dst (dl-load.c:216)
==3943089==    by 0x4006456: _dl_dst_count (dl-load.c:253)
==3943089==    by 0x4006456: expand_dynamic_string_token (dl-load.c:395)
==3943089==    by 0x4006601: fillin_rpath.isra.0 (dl-load.c:483)
==3943089==    by 0x4006932: decompose_rpath (dl-load.c:654)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==  Address 0x1e5613e1 is 1 bytes inside a block of size 8 alloc'd
==3943089==    at 0x4844899: malloc (vg_replace_malloc.c:381)
==3943089==    by 0x4026D8E: malloc (rtld-malloc.h:56)
==3943089==    by 0x4026D8E: strdup (strdup.c:42)
==3943089==    by 0x40068C4: decompose_rpath (dl-load.c:629)
==3943089==    by 0x4008F5F: cache_rpath (dl-load.c:696)
==3943089==    by 0x4008F5F: _dl_map_object (dl-load.c:2114)
==3943089==    by 0x4002504: openaux (dl-deps.c:64)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400293E: _dl_map_object_deps (dl-deps.c:232)
==3943089==    by 0x400C98E: dl_open_worker_begin (dl-open.c:592)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400BE39: dl_open_worker (dl-open.c:782)
==3943089==    by 0x4A2C799: _dl_catch_exception (dl-error-skeleton.c:208)
==3943089==    by 0x400C20B: _dl_open (dl-open.c:886)
==3943089== 
==3943089== 
==3943089== 1 errors in context 5 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6410d78 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 6 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6411d78 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 7 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6412d78 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 8 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6413d78 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 9 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12EB: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6410ce0 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 10 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12E4: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6411ce0 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 11 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6412ce0 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 12 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6413ce0 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 13 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6414ce0 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 14 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD842: ???
==3943089==    by 0x158CCFB5: ???
==3943089==  Address 0x6415ce0 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 15 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12EB: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6411460 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 16 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6411480 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 17 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6412480 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 18 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6413480 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 19 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6414480 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 20 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6415480 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 1 errors in context 21 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6416480 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 22 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6410d18 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 23 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6411d18 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 24 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6412d18 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 25 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==  Address 0x6413d18 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 26 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64112a8 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 27 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64122a8 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 28 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64132a8 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 29 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CD10E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158B2CC5: ???
==3943089==  Address 0x64142a8 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 30 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6410c50 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 31 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64114e0 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 32 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64124e0 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 33 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64134e0 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 2 errors in context 34 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D057C: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6411548 is on thread 2's stack
==3943089==  24576 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 35 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64144e0 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 36 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64154e0 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 37 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CCFB5: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==  Address 0x64164e0 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 38 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6411c50 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 39 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6412c50 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 40 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6413c50 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 41 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C80F9: LinkResolver::resolve_static_call(CallInfo&, LinkInfo const&, bool, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x59C8B2A: LinkResolver::resolve_invoke(CallInfo&, Handle, constantPoolHandle const&, int, Bytecodes::Code, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5742E56: InterpreterRuntime::resolve_invoke(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57433D6: InterpreterRuntime::resolve_from_cache(JavaThread*, Bytecodes::Code) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x158DD834: ???
==3943089==    by 0x158CD0C9: ???
==3943089==  Address 0x6414c50 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 42 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12E4: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6412460 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 3 errors in context 43 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0575: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6412548 is on thread 2's stack
==3943089==  20480 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 4 errors in context 44 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12DD: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6413460 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 4 errors in context 45 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12D6: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6414460 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 4 errors in context 46 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12CF: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6415460 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 4 errors in context 47 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D12C8: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158CD0C9: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09E2A: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==  Address 0x6416460 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 5 errors in context 48 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D056E: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6413548 is on thread 2's stack
==3943089==  16384 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 7 errors in context 49 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0567: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6414548 is on thread 2's stack
==3943089==  12288 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 7 errors in context 50 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0560: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6415548 is on thread 2's stack
==3943089==  8192 bytes below stack pointer
==3943089== 
==3943089== 
==3943089== 7 errors in context 51 of 53:
==3943089== Invalid write of size 4
==3943089==    at 0x158D0559: ???
==3943089==    by 0x158B2CC5: ???
==3943089==    by 0x5749924: JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5724FB0: InstanceKlass::call_class_initializer(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x572618F: InstanceKlass::initialize_impl(JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D09D73: Threads::initialize_java_lang_classes(JavaThread*, JavaThread*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x5D0ABC8: Threads::create_vm(JavaVMInitArgs*, bool*) (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x57F4FB0: JNI_CreateJavaVM (in /usr/lib/jvm/jdk-19/lib/server/libjvm.so)
==3943089==    by 0x48A59EA: JavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x48A9C48: ThreadJavaMain (in /usr/lib/jvm/jdk-19/lib/libjli.so)
==3943089==    by 0x494F401: start_thread (pthread_create.c:442)
==3943089==    by 0x49DD743: clone (clone.S:100)
==3943089==  Address 0x6416548 is on thread 2's stack
==3943089==  4096 bytes below stack pointer
==3943089== 
--3943089-- 
--3943089-- used_suppression:   2084 name valgrind.suppress:2 suppressed: 35,339,504 bytes in 5,673 blocks
==3943089== 
==3943089== ERROR SUMMARY: 118 errors from 53 contexts (suppressed: 60 from 60)
